stellarobservatory/nodes.py

```diff
@@ -27,9 +27,21 @@ def traverse_quorum_set_definition(quorum_set_definition):
         for validator in quorum_set_definition['validators']:
             if validator not in dependencies:
                 dependencies.add(validator)
-                get_node_dependencies(nodes_by_public_key, validator, dependencies)
+                if transitive:
+                    get_node_dependencies(nodes_by_public_key, validator, dependencies)
         for inner_quorum_set_definition in quorum_set_definition['innerQuorumSets']:
             traverse_quorum_set_definition(inner_quorum_set_definition)
 
     traverse_quorum_set_definition(nodes_by_public_key[public_key]['quorumSet'])
     return dependencies
+
+def get_trust_graph(nodes_by_public_key):
+    """
+    Map each node's public key to its trust graph (the set of nodes it references in its quorum slices). 
+    The node itself is excluded.
+    """
+    graph={}
+    for key in nodes_by_public_key:
+        nodes_intransitive = get_node_dependencies(nodes_by_public_key, key, dependencies=set(),transitive=False)
+        graph[key] = set(nodes_intransitive)
```

Comment:
Can you remove `key` from the set? It's included in the dependencies because technically it's in the quorum set but we should ignore it in the trust graph.

Reply:
taken care of in commit  1977dd1e138c1597999b3e909da360c8b1db5715